src/libs/shim-interface/src/lib.rs

```diff
@@ -15,16 +16,60 @@
 //! HTTP request to the server. The server inside shim will multiplex the request
 //! to its corresponding handler and run certain methods.
 
-use std::path::Path;
+use std::fs;
 
-use anyhow::{anyhow, Result};
+use anyhow::{anyhow, Context, Result};
 
 pub mod shim_mgmt;
 
+use kata_sys_util::validate::verify_id;
 use kata_types::config::KATA_PATH;
 
 pub const SHIM_MGMT_SOCK_NAME: &str = "shim-monitor.sock";
 
+fn get_uds_with_sid(short_id: &str, path: &str) -> Result<String> {
+    verify_id(short_id).context("The short id contains invalid characters.")?;
```

Comment:
Ditto.

Reply:
I don't think so. the context of description of the verify_id is what to describe. I will keep it.

Reply:
Does `context("invalid short id")` sound good to you?

Reply:
yeah, partly agree with you. but I don't think "The short id contains invalid characters."doesn't describe what you said. And it is indeed to describe in detail the short id containing invalid char. And IMO, a more detailed message is more useful. So I will keep it here